Source/Core/VideoCommon/TextureInfo.cpp

```diff
@@ -86,6 +86,7 @@ TextureInfo::TextureInfo(const u8* ptr, const u8* tlut_ptr, u32 address,
 
     // load mips - TODO: Loading mipmaps from tmem is untested!
     const u8* src_data = m_ptr + GetTextureSize();
+    tmem_even += m_texture_size;
```

Comment:
We should only do this if `tmem_even` is not a `nullptr`

Reply:
Probably a good idea as a matter of principle, though it doesn't technically matter since we don't dereference in that case.